_articles/zh-hant/how-to-contribute.md

```diff
@@ -204,6 +204,7 @@ redirect_from: /zh-tw/how-to-contribute/
 
 * [GitHub 探索](https://github.com/explore/)
 * [First Timers Only](http://www.firsttimersonly.com/)
+* [你的第一個 PR](https://yourfirstpr.github.io/)
```

Comment:
This link 404s

Reply:
😊

Reply:
@CBID2 did you mean to add this link that 404s?  Could you please remove it?